docs/architecture/metadata/workflow.md

```diff
@@ -0,0 +1,29 @@
+## Live Reloading of a Data Page
+
+When modifying a YAML file for a Data page, executing the `etl ... --grapher` command to visualize updates can be inefficient. The following steps outline a streamlined approach for rapid development and testing.
+
+### Example Step
+For illustration, the step `grapher/gcp/2023-07-10/global_carbon_budget` is considered. The corresponding YAML file, `global_carbon_budget.meta.yml`, is assumed to be edited within the grapher channel and previewed on a [staging Data page](http://staging-site-mojmir/admin/datapage-preview/738081).
+
+### Initial Configuration
+- Add `DEBUG=1` to your `.env.myname` file and any other `.env.*` files in use to expedite ETL operations.
+
+### File Monitoring
+- Execute the ETL command with the `--watch` flag. This monitors the YAML file for changes and automatically re-executes the corresponding step.
+
+### Data Filtering
+- If the dataset contains numerous indicators, consider using `GRAPHER_FILTER=consumption_emissions_per_capita` to filter the data to only relevant variables. This is optional for smaller datasets.
```

Comment:
This is really useful!! But still, the garden step alone takes 19 seconds, so this dataset is not a good example.

Reply:
True. I was using this on grapher steps. I've added a note that you could test your metadata in the grapher step and then move it to garden.